cassdegrees/templates/widgets/rulescripts.html

```diff
@@ -76,10 +76,15 @@
             <label>
                 Students must select...
             </label>
-            <input class="text" onkeydown="javascript: return checkKeys(event)" type="number" v-on:change="check_options" v-model="details.unit_count" min="0" step="6" max="1000" aria-required="true" required>
-            units
         </p>
 
+         <select v-model="details.list_type" required>
+            <option v-for="(msg, type) in list_types" v-bind:value="type">{{ msg }}</option>
+        </select>
+
+<!-- {#       instyle css only temporary #} -->
```

Comment:
Don't merge this one if it can be avoided - move this to a css class if needed.

Reply:
if I move it to style.css, it can't overwrite anu styling (`.anuform fieldset fieldset input` adds padding to the units input box). So I had to settle with the inline styling for the meantime. @JackSekoranja is aware of this decision of mine. But I agree, it 's not ideal.